### ISSUE TITLE
[EN][Achievement] Guru of Drakuru

### ISSUE THREAD

**Issue by xolon** [Atlantiss/BugTracker#4543]
The quest "Cleansing Dark'Tharon" doesn't give credit for the following achievement
http://i.imgur.com/oZPUT0o.jpg
http://i.imgur.com/LBrUZ8t.jpg
Even if credit for achievement gets fixed i can't take quest again, so i request a gm to give me quest again or achievement

**Comment by Beatit** [comment]
yes, 100% true

**Comment by troublemakerr** [comment]
Bump!!!

**Comment by troublemakerr** [comment]
Bump this, very important for people like Xolon who are achievement farmers!!!!

**Comment by HUMBOL** [comment]
BUMP!

**Comment by 1Andas** [comment]
Bump. Still bugged.

**Comment by Samineh** [comment]
I got the credit the problem now is the betrayal quest.

**Comment by Samineh** [comment]
This is fixed just got it today :). The achievement is tricky though. Both quests need to be done the "correct" way.

**Comment by xolon** [comment]
No, there are 2 quests cleansing on the dark tharon, 1 won't give you credit and 1 will, but once you finish 1 you cannot finish the other

**Comment by 1Andas** [comment]
@xolon It happened to me recently, and I can't get the achievement...

**Comment by xolon** [comment]
@1Andas 
I believe there is a solution to fix this.
There are actually 2 of the same quests, 1 at the end of the questline in Grizzly Hills and one inside the dungeon, now one of them doesn't give credit. I happen to get both quests on another servers, dunno if you can acquire both here.

**Comment by 1Andas** [comment]
@xolon Yes I got both quests this is the situation and where problem lies:

1. There are two quests (one is in the dungeon and it's accessible to everyone regardless of their progress in Grizzly Hills area, and the second quest you get from spirit of Drakuru in Grizzly Hills when you summon him near the blazer)
2. The quest in dungeon which is available to everyone triggers the script, you can even finish the quest and get rewards, but it does not give achievement credit (that's what I did).
3. The quest from Grizzly Hills unfortunately does not trigger the event in the dungeon (it didn't for me), so I got in situation where I finished first quest, didn't get achievement, while second quest appears to be broken.
